src/lightly_train/_models/custom/custom_package.py

```diff
@@ -9,42 +9,24 @@
 
 import logging
 from pathlib import Path
-from typing import Any
 
 import torch
 from torch.nn import Module
 
 from lightly_train._models import package_helpers
-from lightly_train._models.model_wrapper import ModelWrapper
-from lightly_train._models.package import Package
+from lightly_train._models.package import BasePackage
 
 logger = logging.getLogger(__name__)
 
 
-class CustomPackage(Package):
+class CustomPackage(BasePackage):
     name = "custom"
 
-    @classmethod
-    def list_model_names(cls) -> list[str]:
-        return []
-
     @classmethod
     def is_supported_model(cls, model: Module) -> bool:
-        return isinstance(model, ModelWrapper)
-
-    @classmethod
-    def get_model(
-        cls, model_name: str, model_args: dict[str, Any] | None = None
-    ) -> Module:
-        raise NotImplementedError()
-
-    @classmethod
-    def get_model_wrapper(cls, model: Module) -> ModelWrapper:
-        if not isinstance(model, ModelWrapper):
-            raise TypeError(
-                "Unsupported model type: Model does not implement FeatureExtractor interface."
-            )
-        return model
+        """Check if the model is supported by this package."""
+        # Custom package supports all models.
+        return isinstance(model, Module)
```

Comment:
```suggestion
        return isinstance(model, ModelWrapper)
```
This must check if the model implements the `ModelWrapper` interface.

Reply:
Hmm doesn't `get_package_from_model` in `package_helpers.py` not work anymore? Because it would then return the `CustomPackage` for any models inheriting `Module` but we actually require models to implement `ModelWrapper`.

Reply:
So, now it should work!